tensorflow/java/src/gen/perl/tftypes.pl

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/perl
```

Comment:
@kbsriram also suggested this might be done in Java. If there is going to be a master program that generates a lot of Java source code, rather than just a simple largely straight-line script like the one that exists now, then writing it in a statically typed language makes a lot of sense to me. I am not yet clear on how this would interact with the Java code that he is working on for ops.